cmd/balance-agent/main.go

```diff
@@ -52,8 +50,7 @@ var (
 	url                 string
 	nodeurl             string
 	// GitRevision is set with build
-	GitRevision = "unknownVersion"
-
+	GitRevision      = "unknownVersion"
 	nodeInfoReported sync.Map
```

Comment:
This is also obsolete, was used as a mutex to not report balance before reporting nodeinfo.

Reply:
We need `GitRevision`, removed `nodeInfoReported`